### PR TITLE
Saras edits

### PR DESCRIPTION
**Pull request by SaraJo**
Updated the analogy
Removed some irrelevant slides
Added images

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -131,48 +124,63 @@ <h3>Step 1: Type an address into the address bar</h3>
 
 				<section>
 					<h3>Step 2: Your computer performs a DNS lookup</h3>
```
Comment (benjamin-hodgson) [inline index.html]: No longer mentioning DNS in the slide itself. How about "your computer looks up the website's IP address"?

File: index.html [hunk 2]
```diff
@@ -131,48 +124,63 @@ <h3>Step 1: Type an address into the address bar</h3>
...
-					<h3>Step 4: The server reponds to your request</h3>
-					<p>The server processes your request and finds the right files. Sometimes it has to "compile" a page from code and other sources, like a database.</p>
-					<aside class="notes">
-						<p>Back-end languages often talk to databases</p>
-					</aside>
-				</section>
+                <section>
+                    <h3>Step 4: Your computer requests information from the server.</h3>
+                    <img src="images/waiter-take-order.jpg" alt="Waiter taking an order" />
+                    <p>You're at the restaurant and your server comes to your table. "What would you like today?"</p>
```
Comment (benjamin-hodgson) [inline index.html]: haha, "server" has more than one meaning. Maybe change to "waiter"?

File: index.html [hunk 3]
```diff
@@ -131,48 +124,63 @@ <h3>Step 1: Type an address into the address bar</h3>
...
+                    <p>"Hello! I would like your finest HyperText Machine Language(HTML) with a side of CSS and JavaScript"</p>
```
Comment (benjamin-hodgson) [inline index.html]: nit pick: HyperText _Markup_ Language

File: index.html [hunk 4]
```diff
@@ -131,48 +124,63 @@ <h3>Step 1: Type an address into the address bar</h3>
...
+                        <p>Back-end languages often talk to databases</p>
+                    </aside>
+                </section>
 
 				<section>
-					<h3>Step 5: The server sends your computer the files</h3>
-					<p>This is usually an HTML page, with some CSS, JavaScript, or media files.</p>
-					<img src="images/mailbox.jpg" alt="Mailbox" />
-					<p><small>Photo credit <a href="http://www.flickr.com/photos/53326337@N00/4384851676">Envelope</a> <a href="http://creativecommons.org/licenses/by-sa/2.0/">cc</a></small></p>
+					<h3>Step 5: The server gets your computer's the request</h3>
```
Comment (benjamin-hodgson) [inline index.html]: nit pick: extra "the"

File: index.html [hunk 5]
```diff
@@ -131,48 +124,63 @@ <h3>Step 1: Type an address into the address bar</h3>
...
+						<h3>Step 6: The server sends its repsonse over HTTP</h3>
+						<p>The server converts its response to HTML/CSS and JavaScript so that your browser can understand it and sends it back to your computer.</p>
+						<img src="images/waiter-delivering.jpg" alt="Waiter delivering food" />
+						<aside class="notes">
+							<p>Each files comes a separate request</p>
+							<p>Requests are packaged into packets</p>
+						</aside>
+					</section>
 				<section>
 					<h3>Step 6: Your browser displays the page</h3>
```
Comment (benjamin-hodgson) [inline index.html]: two step 6s

File: .vs/VSWorkspaceState.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
+  "SelectedNode": "\\index.html",
+  "PreviewInSolutionExplorer": false
+}
```
Comment (benjamin-hodgson) [inline .vs/VSWorkspaceState.json]: accidentally checked in this folder?

**Review by benjamin-hodgson — APPROVED** [review]
Looks good, love the waiter analogy!

File: index.html (hunk 5 above)
Comment (SaraJo) [reply index.html]: Never enough Step 6's

File: index.html (hunk 3 above)
Comment (SaraJo) [reply index.html]: Not a nit pick, accuracy.

**Pull request merged**